src/NovelAudioService.cpp

```diff
@@ -103,7 +101,7 @@ void NovelAudioService::playSound(std::string soundName, int loops) {
   {
     if (Mix_Playing(_channelMap[soundName]) == MIXER_TRUE)
     {
-      std::cout << "Already playing on channel " << _channelMap[soundName] << std::endl;
+      _console.log("Already playing on channel " + _channelMap[soundName], LogLevel::WARN);
```

Comment:
nit: Would be more efficient if this took 2 separate strings and logged them sequentially (as this will allocate and create a new bigger string)